### PR TITLE
add properties management to admin interface

### PR DESCRIPTION
**Pull request by jalehman**
This is a WIP PR--I'll be pushing small achievements to this branch for review.

### CODE REVIEW

File: src/cljs/admin/accounts/events.cljs [hunk 1]
```diff
@@ -271,18 +271,22 @@
...
    {:dispatch [:ui/loading k true]
     :graphql  {:query
-               [[:unit_rate {:unit unit-id
-                             :term term}
-                 [:rate]]]
-               :on-success [::fetch-rate-success k]
+               [[:unit {:id unit-id}
+                 [[:rates [:rate :term]]
+                  [:property [[:rates [:rate :term]]]]]]]
+               :on-success [::fetch-rate-success k term]
```
Comment (queenieho) [inline src/cljs/admin/accounts/events.cljs]: why do we use "::" instead of just one : ?

File: src/cljs/admin/properties/views.cljs [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+  - `active`: the `id` of the active unit
+  - `on-click`: callback that will be passed the id of the clicked unit"
+  [{:keys [units page-size active on-click]
+    :or   {page-size 10, active false}}]
+  (let [state (r/atom {:current 1 :q ""})]
+    (fn [{:keys [units page-size active on-click]
+         :or   {page-size 10, active false}}]
+      (let [{:keys [current q]} @state
+            units'               (->> (drop (* (dec current) page-size) units)
+                                     (take (* current page-size))
```
Comment (queenieho) [inline src/cljs/admin/properties/views.cljs]: what does the asterisk mean?

File: src/cljs/admin/accounts/events.cljs (hunk 1 above)
Comment (jalehman) [reply src/cljs/admin/accounts/events.cljs]: The `::` means "this is a namespaced keyword, where the namespace portion of it is the namespace that the keyword is defined in".

For example:
```clojure
(ns myns)

::keyword ; => :myns/keyword
```

```clojure
(ns myotherns)

::keyword ; => :myotherns/keyword
```

`re-frame` events use a global registry, so if I defined an event handler in one events namespace called `:fetch-rate-success`, and then defined a different event handler in a different namespace called `:fetch-rate-success`, one would overwrite the other. Using `::fetch-rate-success` makes that impossible, unless it's used in the same namespace.

File: src/cljs/admin/properties/views.cljs (hunk 2 above)
Comment (jalehman) [reply src/cljs/admin/properties/views.cljs]: multiplication

Comment (queenieho) [reply src/cljs/admin/properties/views.cljs]: that makes sense...

**Pull request merged**